_content/av/guardian-deity-fudo-myoo.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "The Buddhist Guardian Deity: Fudo Myoo"
+authors:
+  - "Asian Art Museum"
```

Comment:
At 2:54 in the video you'll see the writer. Make them the author. And at 2:59 you'll see the editor.